### ISSUE TITLE
Feature: remove footer from Workspaces

### ISSUE THREAD

**Issue by isabensusan** [open-sauced/app#2948]
### Type of feature

🍕 Feature

### Current behavior

We currently show the footer everywhere in the app, but it can sometimes interfere with the layout without even being too useful.

<img width="1348" alt="image" src="https://github.com/open-sauced/app/assets/27322879/15a0dc2d-0b4f-4ae9-a9ba-659d6098450d">

### Suggested solution

Remove the footer for a cleaner look, but **only for the Workspaces paths**. 

So:
| Path | Footer? |
| --- | --- |
| Explore Page | YES |
| Highlights Feed | YES |
| Workspaces | NO |
| Insight pages | NO |

<img width="1512" alt="image" src="https://github.com/open-sauced/app/assets/27322879/8aec8cc3-9ef3-4341-b96b-fadef9770cac">

## Research behind this decision

**Will this hurt our SEO?**
tl;dr: It shouldn't. 
- Workspaces are a web-based application/tool, and best practices dictate that it's ok to not have a footer on web apps. 
- The footer rule applies mostly to content-based pages, in which we would still leverage the footer (landing pages, highlight feed, explore page, etc.). 
- Our current Workspaces layout isn't optimized for SEO anyways, so it's not like it's doing us any favors in that front nyways, but it is hurting usability.

A quick benchmark of other tools confirmed that most webapps don't use footers:
| App | Footer? |
| --- | --- |
| Supabase | NO |
| Notion | NO |
| Posthog | NO |
| Analytics | NO |

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

**Comment by takanome-dev** [comment]
cc @isabensusan, does this need some discussion with the team or is it ready to be worked on?

**Comment by isabensusan** [comment]
hey @takanome-dev ! We got buy-in from the team, so if you'd want to take this issue, go for it! :)
